### DISCUSSION TITLE
Taking long time to display report

### DISCUSSION THREAD

**Discussion by ravindraP20** [allure-framework/allure2#2504] [Questions & Support] [answered]
Hi Team,

I am using below task in my azure devops pipeline.

        - task: PublishAllureReport@1
          displayName: Allure Report        
          inputs:
            allureVersion: 2.27.0
            testResultsDir: ./WDI5/allure-results

As of now we have test result of just 52 test cases, Below is the log of the task:

Downloading Allure CLI from https://github.com/allure-framework/allure2/releases/download/2.27.0/allure-2.27.0.tgz
Downloading: https://github.com/allure-framework/allure2/releases/download/2.27.0/allure-2.27.0.tgz
Caching tool: allure 2.27.0 x64
Prepending PATH environment variable with directory: /home/vsts/work/1/s/0f029fe0-dceb-4b68-b1fe-6b1a5cccdc77/allure-2.27.0/bin
/home/vsts/work/1/s/0f029fe0-dceb-4b68-b1fe-6b1a5cccdc77/allure-2.27.0/bin/allure generate --single-file --name Allure Report --clean -o /home/vsts/work/1/s/4250223a-3759-40e3-bc18-3ad44dd2c016 /home/vsts/work/1/s/WDI5/allure-results
Report successfully generated to /home/vsts/work/1/s/4250223a-3759-40e3-bc18-3ad44dd2c016
Report file is too large (446.59371757507324MB), uploaded in 45 chunks.

It takes log time (3-5 minutes) to display the report. For the first time it even takes more time. I am afraid as we are going to add many test cases and it may take more longer time to display the report.
![image](https://github.com/allure-framework/allure2/assets/72969399/cf8941bd-3090-4ec6-97c7-54e6a57c6640)

I have tried to generate the report using allure generate --single-file allure-results --clean but the above task does not support the command. I thought of generating the report and publishing it as artifact in the pipeline using PublishBuildArtifacts@1 task. But everytime it generates the report to a random UUID directory.

Is there any way to reduce the time it takes to display the report or to publish the index.html file as pipeline artifact so that the report can be opened in seconds.

Regards,
Ravindra

**Comment by baev** [comment, marked as answer]
The issue is that the report is 447MB, so it takes some time to download and open in the browser. The problem is most likely with excessive amounts of attachments.

**Comment by Shekur-Kariz** [comment]
Hi all. I am having the same issue. I tried to run only a few tests in regression suite but still taking 1 - 2 hours to upload the results to Github.
